src/espIdf/debugAdapter/debugAdapterManager.ts

```diff
@@ -24,14 +24,19 @@ import * as idfConf from "../../idfConfiguration";
 import { Logger } from "../../logger/logger";
 import { appendIdfAndToolsToPath, isBinInPath, PreCheck } from "../../utils";
 import { getProjectName } from "../../workspaceConfig";
+import { EOL, tmpdir } from "os";
+import { outputFile } from "fs-extra";
 
 export interface IDebugAdapterConfig {
+  coreDumpFile: string;
```

Comment:
Shouldn't this be optional, else we need to provide the `coreDumpFile` everywhere?

Reply:
Yes, might have been a typo.